src/components/MainView.js

```diff
@@ -9,13 +12,49 @@ export default class MainView extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
+      data: this.import(csvdata)
     };
   }
 
+  import(csv) {
+    let results = Papa.parse(csv, {
+      header: true,
+      dynamicTyping: true
+    });
+
+    if (results.errors.length) {
+      results.errors.forEach((err) => console.error(err));
+    }
+
+    return results.data;
+  }
+
+  export(json) {
```

Comment:
can we  just use `this.state.data` instead of having to pass around `json`? seems simpler